whatis277/harvest
# bean: Add app controller

Moves home handler to app controller

Duplicates `handler/shared` to `controller/app/shared` which will be used by payment methods and subscriptions

Testing instructions:
1. `dc up --build bean`
2. Ensure there are no errors in the logs
3. Visit http://localhost:8080
4. Visit http://localhost:8080/home
5. Ensure both load fine
